src/db/util.py

```diff
@@ -88,7 +88,8 @@ def get_internal_srid(crs):
     return srid
 
 
-def get_crs(srid, conn_cur=None, *, use_internal_srid=True):
+def get_crs(srid, conn_cur=None, *, use_internal_srid=None):
```

Comment:
I do not understand, why we do not use `def get_crs(srid, *, use_internal_srid, conn_cur=None):`